admin/app/components/login-form.js

```diff
@@ -13,7 +13,11 @@ export default class LoginForm extends Component {
   @tracked email;
   @tracked password;
   @tracked errorMessage;
+  @service oidcIdentityProviders;
 
+  get isGoogleIdentityProviderEnabled() {
+    return this.oidcIdentityProviders.isProviderEnabled('google');
+  }
```

Comment:
Juste ajouter un petit espacement.

```suggestion
  }

```